cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,25 @@
+class IframePage {
+    private testPage: string;
+    private iframe: string;
+    private title: string;
+  
+    constructor() {
+      this.testPage = 'https://www.w3schools.com/html/html_iframe.asp';
+      this.iframe = '[src="default.asp"]';
+      this.title = 'a[title="CSS Tutorial"]';     
+    }
+
+    public visit(): void{
+        cy.visit(this.testPage);
+    }
+
+    public getFrameTitle(tittlePage: string): void{
+        return cy.iframe(this.iframe).should("contain.text", tittlePage);
+    }
```

Comment:
as we are asking you to get the frame title, in this case is required to not perform a validation, instead of that you should return the frame title

Reply:
resolved